### PR TITLE
Fix getting started use of webviews

### LINKED ISSUE #133755: Walkthroughs not allowing clicks in detail pane
Open any walkthrough that has click actions in the detail pane (theme switching, extensions, video, etc.) and nothing appears to work. There are some errors in dev tools when clicking the video link:

CleanShot 2021-09-24 at 09 40 58@2x

### PR DESCRIPTION
**Pull request by JacksonKearl**
Fix #133755

cc @mjbvz not sure why this change is needed now, this is regarding the SVG rendering, which does not go through dompurify.

### CODE REVIEW

File: src/vs/workbench/contrib/webview/browser/webviewElement.ts [hunk 1]
```diff
@@ -370,7 +370,7 @@ export class IFrameWebview extends Disposable implements Webview {
 		const element = document.createElement('iframe');
 		element.name = this.id;
 		element.className = `webview ${options.customClasses || ''}`;
-		element.sandbox.add('allow-scripts', 'allow-same-origin', 'allow-forms', 'allow-pointer-lock', 'allow-downloads');
+		element.sandbox.add('allow-scripts', 'allow-same-origin', 'allow-forms', 'allow-pointer-lock', 'allow-downloads', 'allow-popups');
```
Comment (JacksonKearl) [inline src/vs/workbench/contrib/webview/browser/webviewElement.ts:373]: seems that there are actually two `iframes`? I think this is relatively new? But the outer `iframe` has all the sandbox permissions granted always, the inner one has the conditional on the passed options.

Comment (mjbvz) [reply src/vs/workbench/contrib/webview/browser/webviewElement.ts:373]: Yes that's correct. The outer iframe is the one we manage while the inner one has content from extensions

**Review by mjbvz — APPROVED** [review]

**Comment by mjbvz** [comment]
Oh actually, can you see if changing the link's [target](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target) fixes this? 

This was working before right? I wonder if the dom purify sanitization changed something

**Comment by JacksonKearl** [comment]
The SVG's don't go through dompurify, so I really don't know what changed here. With this `http` links start working again, but `command` links are still broken.

**Comment by JacksonKearl** [comment]
Not sure how the `target` would be set for links in SVG's...

**Comment by JacksonKearl** [comment]
No longer needed with 8e35bdb33bfff18881cd6dc68945d0a59d979361

**Pull request closed without merging**